### ISSUE TITLE
Can't create or load IronPython project

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#1110]
PTVS 3.0

Problem creating a new IronPython project, when IronPython is already installed.  I get a message box:
```
---------------------------
Microsoft Visual Studio
---------------------------
The domain manager specified by the host could not be instantiated.
---------------------------
OK   
---------------------------
```

You can create one if you don't have IronPython installed.  But if I try to load it after installing IronPython, I get the same error in output window.

**Comment by zooba** [comment]
I think this is because of some GAC changes and we don't have enough search paths added to the new appdomain we create for IronPython IntelliSense.

**Comment by zooba** [comment]
Should be fixed with #1200 and #1216
